boilerplates/sentry/files/pages/sentry/+Page.vue

```diff
@@ -0,0 +1,35 @@
+/*{ @if (it.BATI.has("vue")) }*/
```

Comment:
Move into `vue-sentry`

Reply:
I have tried to keep everything in one place to make global changes easier but will follow your advices 😄